ubuntu/16.04/usr/local/share/container/baseimage-10.sh

```diff
@@ -25,3 +31,20 @@ check_development_start() {
 do_development_start() {
   :
 }
+
+do_build_user_ssh_keys() {
+  set +x
+  if [ -n "$BUILD_USER_SSH_PRIVATE_KEY" ] && [ -n "$BUILD_USER_SSH_PUBLIC_KEY" ]; then
+    echo "Setting up SSH keys for the build user"
+    mkdir -p /home/build/.ssh/
+    echo "$BUILD_USER_SSH_PRIVATE_KEY" | base64 --decode > /home/build/.ssh/id_rsa
+    echo "$BUILD_USER_SSH_PUBLIC_KEY" | base64 --decode > /home/build/.ssh/id_rsa.pub
+    echo "$BUILD_USER_SSH_KNOWN_HOSTS" | base64 --decode > /home/build/.ssh/known_hosts
```

Comment:
I've no clue how to do that and neither what it means 😛 Would you mind taking over this PR @andytson-inviqa? @kierenevans is not really available until next Monday and it's kind of a blocker for a project...